packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -385,6 +385,11 @@ class BuildableIOSApp extends IOSApp {
   String get archiveBundlePath
     => globals.fs.path.join(getIosBuildDirectory(), 'archive', globals.fs.path.withoutExtension(_hostAppBundleName));
```

Comment:
slightly tangential: at some point we might want to take stock of all the "temporary but user accessible" build artifacts mixed with build intermediaries in the build/ folder and organize them. 

Right now I have 

```
build
├── 3f44400bd8f8c7820069645ed94a731d
├── app
│   ├── generated
│   │   ├── ap_generated_sources
│   │   ├── not_namespaced_r_class_sources
│   │   ├── res
│   │   └── source
│   ├── intermediates
│   │   ├── annotation_processor_list
│   │   ├── apk_list
│   │   ├── blame
│   │   ├── bundle_manifest
│   │   ├── check_manifest_result
│   │   ├── compatible_screen_manifest
│   │   ├── dex
│   │   ├── duplicate_classes_check
│   │   ├── external_file_lib_dex_archives
│   │   ├── external_libs_dex
│   │   ├── flutter
│   │   ├── generated_proguard_file
│   │   ├── incremental
│   │   ├── instant_app_manifest
│   │   ├── javac
│   │   ├── lint_jar
│   │   ├── manifest_merge_blame_file
│   │   ├── merged-not-compiled-resources
│   │   ├── merged_assets
│   │   ├── merged_java_res
│   │   ├── merged_jni_libs
│   │   ├── merged_manifests
│   │   ├── merged_native_libs
│   │   ├── merged_shaders
│   │   ├── metadata_feature_manifest
│   │   ├── processed_res
│   │   ├── proguard-files
│   │   ├── proguard-rules
│   │   ├── res
│   │   ├── res_stripped
│   │   ├── shader_assets
│   │   ├── signing_config
│   │   ├── stripped_native_libs
│   │   ├── symbols
│   │   ├── transforms
│   │   └── validate_signing_config
│   ├── outputs
│   │   ├── apk
│   │   ├── flutter-apk
│   │   ├── logs
│   │   └── mapping
│   ├── reports
│   └── tmp
│       ├── compileDebugJavaWithJavac
│       ├── compileProfileJavaWithJavac
│       ├── compileReleaseJavaWithJavac
│       ├── packLibsflutterBuildDebug
│       ├── packLibsflutterBuildProfile
│       └── packLibsflutterBuildRelease
├── fa37e11e55641aa14c7c94ec233f11be
└── ios
    ├── Debug-iphoneos
    │   └── Runner.app
    ├── Profile-iphoneos
    │   ├── Runner.app
    │   └── Runner.app.dSYM
    ├── Runner.build
    │   └── Release-iphoneos
    └── iphoneos
        └── Runner.app
```

without counting ios-frameworks, aars etc. We don't have to fix anything here but it's a question for the next comment.